voice-change-o-matic/scripts/app.js

```diff
@@ -307,26 +292,24 @@ function init() {
   }
```

Comment:
Line 271, 274, 277, 280: use arrow functions.

Reply:
I think we can use method shorthand here even:

```js
    return {
      apply() {
        wetNode.gain.setValueAtTime(0.75, audioContext.currentTime);
      },
      discard() {
        wetNode.gain.setValueAtTime(0, audioContext.currentTime);
      },
      isApplied() {
        return wetNode.gain.value > 0;
      },
      placeBetween(inputNode, outputNode) {
        inputNode.connect(delay);
        delay.connect(wetNode);
        wetNode.connect(filter);
        filter.connect(delay);

        inputNode.connect(dryNode);
        dryNode.connect(mixer);
        wetNode.connect(mixer);
        mixer.connect(outputNode);
      },
```